### DISCUSSION TITLE
"maximun source-receiver distance" Parameter

### DISCUSSION THREAD

**Discussion by IsotoCedex** [Universite-Gustave-Eiffel/NoiseModelling#518] [Q&A] [answered]
Hi,
**Description**
I have run a large area model with the following imput data:
- Buildings with heiht data
- Roads, and LW_ROADS calculated from traffic
- DEM, with hight
- Receivers, with value HEIGHT = 0
- maximun source-receiver distance = 3500

Althought it is a large model (the entire surface of Spain), the computation has been really fast (2 hours).

**Results**
But results are not satisfactory for the following reasons:
- Receivers located on roads (intersecting roads) have inconsistent values. For example, if the road emission is 86 dB, the reveiver on the road  has LDEN_RESULT = 16 dB(A).
- Strange behaviour of noise depending on the distance from the source. Also receivers close to the source have lower values than other further away.

**Questions/doubts**:
- Is it necessary that the data of the RECEIVERS have the height value of DTM?
- Does ROADS feature need to have Z data?
- Does NM place roads on the ground if DEM data is provided?
- How does the parameter "maximum source-receiver distance" influence the speed of calculation?
- Does it influence the accuracy of the results?

![image](https://user-images.githubusercontent.com/87642224/192466485-28dc747a-225a-4484-9f20-6bdd88b5ccb7.png)

**Comment by nicolas-f** [comment]
Hi,

Height of receivers/sources are by default relative to the dtm. So for receivers you should at least set an height > 0 (Cnossos is 4m height and 2m of facades). For road, [Road_Emission_from_Traffic](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/blob/master/wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/NoiseModelling/Road_Emission_from_Traffic.groovy) will create source at 0.05m from above the surface.

maximum source-receiver distance will stop searching for noise source after the specified distance, so it will greatly impact the computation time. You should set 350m to this value, not more than 800m as this is out of the bounds of cnossos model.

![image](https://user-images.githubusercontent.com/1382241/192510366-bda093ac-7485-410e-825c-aca1859d6b09.png)
_Methods for the Calculation of Sound Propagation
Wolfgang Probst_

**Reply by IsotoCedex** [reply]
Hi,

So, regarding the questions I asked:

**Is it necessary that the data of the RECEIVERS have the height value of DTM?**
- Yes. must be 4 m for CNOSSOS Method
- But, in the documentation example, the Z value in RECEIVERS is  "0.00". So I assume that NM applies 4 m for the calculation.

```
{
"type": "FeatureCollection",
"name": "receivers",
"crs": { "type": "name", "properties": { "name": "urn:ogc:def:crs:EPSG::2154" } },
"features": [
{ "type": "Feature", "properties": { "PK": 1 }, "geometry": { "type": "Point", "coordinates": [ 223495.988041148491902, 6757167.989008219912648, 0.0 ] } },
{ "type": "Feature", "properties": { "PK": 2 }, "geometry": { "type": "Point", "coordinates": [ 223545.988041148491902, 6757167.989008219912648, 0.0 ] } },
{ "type": "Feature", "properties": { "PK": 3 }, "geometry": { "type": "Point", "coordinates": [ 223595.988041148491902, 6757167.989008219912648, 0.0 ] } },
....
```


**Does ROADS feature need to have Z data?**
- The answer is no, because the "Road_emission_from_traffic" module puts a default value "0,05"

**Does NM place roads on the ground if DEM data is provided?**
- Yes

**How does the "maximum source-receiver distance" parameter  influence the speed of calculation?**
- I don´t understand why, the more I increase the "maximum source-receiver distance", the less time it takes to calculate.
- If the source search from receivers ends within the "maximum source-receiver distance", it should take less time with smaller distances, right?

**Does the  "maximum source-receiver distance" parameter influence the accuracy of the results?**
- It should be below 800. Recommended 350.
- **But, what if I want to calculate isophones of 30 dB(A) and below?**




Thank you

**Comment by pierromond** [comment, marked as answer]
> So I assume that NM applies 4 m for the calculation.

No, but you can use the WPS block set_height, to change the height of you receivers. It is a mistake in the documentation, we will change it soon, thank you !

> I don´t understand why, the more I increase the "maximum source-receiver distance", the less time it takes to calculate. If the source search from receivers ends within the "maximum source-receiver distance", it should take less time with smaller distances, right?

It depends, if you distance is very small, the domain is cutted in a lot of small pieces to run your calculation in parrallel. The preparation of the parallel computing can be in some few cases > at the path finding algorithm computation time

>But, what if I want to calculate isophones of 30 dB(A) and below?

You will have to go further than the limitation... For a project we run CNOSSOS over 2.5km, it is far from the "official" limits of the model, but still will give you some coherent sound levels. You have to be aware that further than 800m some physicial phenomenom can be predominant (link to meteorology for example) and are not well describe in the CNOSSOS model at the distances.
